server/src/app.ts

```diff
@@ -102,7 +104,47 @@ app.use('/api/summarizeTranscript', cors(), summarizeTranscript);
  */
 app.use('/api/notificationEvents', cors(), notificationEvents);
 
+/**
+ * route:/fetchTranscriptionState
+ * purpose: Fetch the transcription state for a call
+ */
+app.use('/api/fetchTranscriptionState', cors(), fetchTranscriptState);
+
 const config = getServerConfig();
+/**
+ * route: wss://<host>/
+ * purpose: WebSocket endpoint to receive transcription events
+ *
+ * Don't forget to secure this endpoint in production
+ * https://learn.microsoft.com/en-us/azure/communication-services/how-tos/call-automation/secure-webhook-endpoint?pivots=programming-language-javascript
+ */
+console.log('WebSocket server port:', config.callAutomation?.ServerWebSocketPort);
```

Comment:
I think this is removed in a later PR but if not remove this

Reply:
yes in the end there is no port and is instead integrated into the express server.